common-services.yml

```diff
@@ -1,4 +1,13 @@
 services:
+  core:
+    image: audius/core:${TAG:-current}
+    restart: unless-stopped
+    ports:
```

Comment:
Also do we want to add the performance restrictions here? Memory and cpu? Maybe it's to see just how much it takes up